core/src/main/java/com/graphhopper/routing/util/FootFlagEncoder.java

```diff
@@ -331,6 +333,10 @@ protected int handlePriority(ReaderWay way, int priorityFromRelation) {
         else
             weightToPrioMap.put(110d, priorityFromRelation);
 
+        if (isWayAccessDestination(way))
```

Comment:
We could merge this and the above `isWayAccessDestination`. But then we would change the Priority outside of the `handlePriority` method, which I don't like that much. I can change it if you would prefer that?

Reply:
This looks good to me.